recipes/zlib/all/conanfile.py

```diff
@@ -29,7 +29,12 @@ class ZlibConan(ConanFile):
 
     @property
     def _is_mingw(self):
-        return self.settings.os == "Windows" and self.settings.compiler == "gcc"
+        host_os = self.settings.os
+        host_compiler = self.settings.get_safe("compiler")
+        mingw_gcc = host_os == "Windows" and host_compiler == "gcc"
+        mingw_clang = host_os == "Windows" and host_compiler == "clang" and \
```

Comment:
Could you please added a comment explaining why `not` that comes next is important for futures contributors who might not know I'd like to try avoiding breaking this again

Reply:
Maybe just add a link to https://github.com/conan-io/conan/pull/12678

Reply:
Sure, "for more details  see..." it good